<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualisationBDD</class>
 <widget class="QWidget" name="VisualisationBDD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualiseur de données</string>
  </property>
  <widget class="QTableWidget" name="vueTable">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>691</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Choix de la base de données :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>425</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Ligne de commande SQL :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputSQL">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>691</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonExec">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>480</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Éxécuter</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonEffacer">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>480</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Effacer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputPath">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonSelection">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>36</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Sélectionner</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="vueArborescence">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>521</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
